src/index.ts

```diff
@@ -27,6 +28,7 @@ export interface Logger<T> {
     success(content: T): Promise<void>;
     warn(content: T): Promise<void>;
     error(content: T): Promise<void>;
+    logMultiple(content: MultipleLogContent): Promise<void>;
```

Comment:
we can think about the best name for this. Is batchLog() better than logMultiple? Low importance